framework/helpers/BaseHtml.php

```diff
@@ -1154,6 +1154,9 @@ public static function activeHint($model, $attribute, $options = [])
      * - header: string, the header HTML for the error summary. If not set, a default prompt string will be used.
      * - footer: string, the footer HTML for the error summary.
      * - encode: boolean, if set to false then the error messages won't be encoded.
+     * @since 2.0.10
+     * - showAllErrors: boolean, if set to true every error message for each attribute will be shown otherwise
```

Comment:
```
- showAllErrors: boolean, if set to true, all error messages for all the attributes will be shown. Otherwise only the first error message for each attribute will be shown. Option is available since 2.0.10
```

The previous line should be removed